src/main/java/frc/robot/Constants.java

```diff
@@ -42,6 +45,9 @@ public final class Constants {
    */
   public static final double MAX_VOLTAGE = 12;
 
+  public static final Transform3d ROBOT_TO_BUMPERS = new Transform3d(0, 0, Units.Meters.convertFrom(0, Units.Inches),
```

Comment:
Lets default to 2025 bot
```suggestion
  public static final Transform3d ROBOT_TO_BUMPERS = new Transform3d(0, 0, Units.Meters.convertFrom(5, Units.Inches),
```